.github/workflows/build.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.5, 3.6, 3.7, 3.8, 3.9, pypy-3.7]
+        python-version: [3.8, 3.9, "3.10"]
```

Comment:
Let's only keep the mentioned version and add only support for `Django` for now.

Reply:
we would need to drop Django main from tests as 3.10 is required for it.